server/chat-plugins/room-faqs.ts

```diff
@@ -133,6 +140,46 @@ export const commands: ChatCommands = {
 	],
 };
 
+export const pages: PageTable = {
+	roomfaqs(args, user) {
+		const room = this.requireRoom();
+		this.title = `[Room FAQs]`;
+		// allow it for users if they can access the room
+		if (!user.inRooms.has(room.roomid) && room.settings.isPrivate && !user.isStaff) {
```

Comment:
```suggestion
		if (!room.checkModjoin(user)) {
```

Reply:
Is this a new function? I'm confused why this wasn't used when you added quotes.

Reply:
No, it's not. Bluntly, I forgot it existed. I'm going to PR it to quotes later.